lib/yoti/activity_details.rb

```diff
@@ -45,7 +45,7 @@ def profile
       Yoti::Profile.new(@extended_profile)
     end
 
-    protected
+    private
```

Comment:
Even though we can be almost certain that no one will be using this method, I'm thinking it might be safer to wait until the major release to change this. What do you think?

Reply:
I've now reverted this change